packages/brain/src/index.ts

```diff
@@ -88,19 +100,26 @@ await brainDb.initialize(signerApi, validatorApi);
 logger.debug(brainDb.data);
 
 // CRON
-export const cron = new Cron(
+export const reloadValidatorsCron = new CronJob(
   60 * 1000,
-  signerApi,
-  signerUrl,
-  validatorApi,
-  brainDb
+  new ReloadValidators(
+    signerApi,
+    signerUrl,
+    validatorApi,
+    brainDb
+  ).reloadValidators
+);
+reloadValidatorsCron.start();
+const proofOfAttestationCron = new CronJob(shareCronInterval, () =>
```

Comment:
Why here attestation and below validation? I would be consistent with the naming